openlibrary/catalog/add_book/__init__.py

```diff
@@ -742,13 +742,13 @@ def load(rec, account=None):
         e['ocaid'] = rec['ocaid']
         need_edition_save = True
 
+    # Add list fields to edition as needed
     edition_fields = [
-        'local_id', 'ia_box_id', 'ia_loaded_id', 'source_records']
```

Comment:
Does that mean that up to now, this code has been adding `ia_box_id` and `ia_loaded_id` fields to OL editions?

Reply:
@cdrini  If importbot was populating that data from archive.org records. I'm not sure it was though. This looks like a relic from early imports. Recent archive.org imports come from the MARC records, which don't know about `ia_box_id`.